code/WorkInProgress/JobXPRewards.dm

```diff
@@ -383,6 +383,40 @@ mob/verb/checkrewards()
 			src.claimedNumbers[usr.key] --
 			return
 
+
+//Head of Personal
+
+/datum/jobXpReward/colorfulID
+	name = "Blue ID briefcase"
+	desc = "Trade out your plain brown briefcase for a colorful blue one! BE SURE TO REMOVE CARDS BEFORE CLAIMING THIS, IT WILL DESTROY THOSE INSIDE!"
```

Comment:
I don't like that IDs can be destroyed by this. Has a bit of potential for general nastiness.

I would just use the blue one as the base one and remove this job reward stuff for now.

Reply:
alright